<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageTableWindow</class>
 <widget class="QMainWindow" name="ManageTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageTableWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_code">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_content">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>增加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>391</width>
      <height>451</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>编码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>内容</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_view">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查看</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
